defaults/defaults.go

```diff
@@ -135,6 +137,9 @@ const (
 	// ClustermeshMaxConnectedClusters is the default number of the maximum
 	// number of clusters that should be allowed to connect to the Clustermesh.
 	ClustermeshMaxConnectedClusters = 255
+
+	// Default timeout for Connectivity Test Suite (disabled by default)
```

Comment:
I think it's fine to leave the default timeout disable by default. Otherwise users outside of GHA might easily hit it, depending on their infrastructure.